tests/test_help_messages.py

```diff
@@ -30,10 +73,10 @@ def _iter_through_commands(command, path):
 
 @pytest.mark.parametrize(
     "command",
-    iter_through_all_commands_paths(),
-    ids=(".".join(cmd) for cmd in iter_through_all_commands_paths()),
+    iter_through_all_commands_and_subcommands_paths(),
```

Comment:
This should remain `iter_through_all_commands_paths`, otherwise both tests are essentially the same